docs/list.yaml

```
@@ -50,6 +50,42 @@ PHP:
       description: Adds CORS headers support in your Laravel application
       url: https://github.com/barryvdh/laravel-cors
       tags: ['Laravel', 'CORS']
+    barryvdh/laravel-debugbar:
+      description: Laravel Debugbar (Integrates PHP Debug Bar)
+      url: https://github.com/barryvdh/laravel-debugbar
+      tags: ['Laravel', 'Debugging']
+    jenssegers/mongodb:
+      description: A MongoDB based Eloquent model and Query builder for Laravel (Moloquent)
+      url: https://github.com/jenssegers/laravel-mongodb
+      tags: ['Database', 'Moloquent']
+    robmorgan/phinx:
+      description: Simple PHP Database Migrations
+      url: https://github.com/cakephp/phinx
+      tags: ['Migrations']
+    phpoffice/phpspreadsheet:
+      description: A pure PHP library for reading and writing spreadsheet files
+      url: https://github.com/jenssegers/laravel-mongodb
+      tags: ['Office']
+    spatie/laravel-activitylog:
+      description: Log activity inside your Laravel app https://docs.spatie.be/laravel-activitylog
+      url: https://github.com/spatie/laravel-activitylog
+      tags: ['Laravel', 'Activity Log', 'Logging']
```

As well, I think ```Activity Log``` tag is unlikely to repeat again.
```suggestion
      tags: ['Laravel', 'Logging']
```